package.json

```diff
@@ -20,6 +20,6 @@
     "frontend-build:development" : "./node_modules/gulp/bin/gulp.js build:development",
     "frontend-build:production" : "./node_modules/gulp/bin/gulp.js build:production",
     "frontend-build:watch" : "./node_modules/gulp/bin/gulp.js watch",
-    "postinstall": "npm run frontend-install"
+    "postinstall": "npm run frontend-install && npm run frontend-build:production"
```

Comment:
I'd keep the `package.json` files the same between the frontend apps, I don' think any of the others are running the build on install